egui_glow/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to the `egui_glow` integration will be noted in this file.
 
 
 ## Unreleased
+* Add `workarounds` to `Painter::new` to pass GLSL macro to fix platform specific behavior.
```

Comment:
```suggestion
```

Since this isn't a generally usable thing (you need to also modify the shaders), I think we can omit it from this changelog